webrecorder/browsertrix
# Fix crawl list action menu positioning

Fixes https://github.com/webrecorder/browsertrix-cloud/issues/1111 (also reported in https://github.com/webrecorder/browsertrix-cloud/issues/1346)



### Changes

Refactors `btrix-crawl-list` dropdown action menu to use `sl-dropdown` auto-positioning to fix menu clipping.

### Manual testing

1. Log in as user with crawler permissions and go to "Archived Items"
2. Click "..." action menu button. Verify menu is visible in all wide viewport sizes
3. Regression test:
  - Clicking archived item takes you to detail page
  - Clicking each menu item works as expected
4. Go to "Crawling" page and click into a workflow with items
5. Click "..." in item. Verify menu opens without clipping
6. Log out and log back in as user with viewer permissions
7. Go to "Crawling" page and click into a workflow with items. Verify that "..." action menu button is hidden

### Demo


https://github.com/webrecorder/browsertrix-cloud/assets/4672952/813c08a3-50e9-4ba7-a65e-20c9ecde451e
